run-integration-tests.py

```diff
@@ -196,6 +196,43 @@ def run_s3_log_store_util_integration_tests():
         raise
 
 
+def run_iceberg_integration_tests(root_dir, version, spark_version, iceberg_version, use_local):
+    print("\n\n##### Running Iceberg tests on version %s #####" % str(version))
+    clear_artifact_cache()
+    if use_local:
+        run_cmd(["build/sbt", "publishM2"])
+
+    test_dir = path.join(root_dir, path.join("delta-iceberg", "integration_tests"))
+    files_to_skip = {"using_with_pip.py", "missing_delta_storage_jar.py", "image_storage.py"}
+
+    test_files = [path.join(test_dir, f) for f in os.listdir(test_dir)
+                  if path.isfile(path.join(test_dir, f)) and
+                  f.endswith(".py") and not f.startswith("_") and
+                  f not in files_to_skip]
```

Comment:
given there is only one test, just use that directly than listing everything and excluding.

Reply:
I think this allows us to add other iceberg integration suites in the same directory as well, so it's ok to have this way?

Reply:
When that happens, we can add the test to `test_files` list here.

Reply:
alright, i don't have strong opinion on this